app/src/main/java/com/bitchat/android/ui/ChatHeader.kt

```diff
@@ -264,8 +263,8 @@ fun ChatHeaderContent(
             Log.d("ChatHeader", "Header recomposing: peer=$selectedPrivatePeer, isFav=$isFavorite, sessionState=$sessionState")
             
             // Pass geohash context and people for NIP-17 chat title formatting
-            val selectedLocationChannel by viewModel.selectedLocationChannel.observeAsState()
-            val geohashPeople by viewModel.geohashPeople.observeAsState(emptyList())
+            val selectedLocationChannel by viewModel.selectedLocationChannel.collectAsState()
+            val geohashPeople by viewModel.geohashPeople.collectAsState()
```

Comment:
same as https://github.com/permissionlesstech/bitchat-android/pull/518/files#r2574005584 
```suggestion
            val selectedLocationChannel by viewModel.selectedLocationChannel.collectAsStateWithLifecycle()
            val geohashPeople by viewModel.geohashPeople.collectAsStateWithLifeCycle()
```